src/js/app.js

```
@@ -1,24 +1,104 @@
+    const table = document.getElementById("achievements-table");
+    const tbody = document.getElementById("achievements-body");
+    tbody.innerHTML = "";
+
+    // Convert achievements to array and sort by threshold
+    const achievementsArray = Object.entries(achievements)
+        .map(([name, achievement]) => ({ name, ...achievement }))
+        .sort((a, b) => a.threshold - b.threshold);
+
+    // Find the last earned achievement and next two unearned
+    let lastEarned = null;
+    let nextUnearned = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'nextUnearned' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)